frontends/web/src/components/icon/icon.tsx

```diff
@@ -156,6 +157,8 @@ export const Save = (props: ImgProps) => (<img src={saveSVG} draggable={false} {
 export const SaveLight = (props: ImgProps) => (<img src={saveLightSVG} draggable={false} {...props} />);
 export const Star = (props: ImgProps) => (<img src={starSVG} draggable={false} {...props} />);
 export const StarInactive = (props: ImgProps) => (<img src={starInactiveSVG} draggable={false} {...props} />);
+// check component for cusotm multi-dropdown select
```

Comment:
comment not needed IMO, this icon could also be used in a different component in the future.